### DISCUSSION TITLE
What's the workflow I should use to find notes by title?

### DISCUSSION THREAD

**Discussion by EddieBrunstedt** [epwalsh/obsidian.nvim#554] [Q&A] [answered]
_(I feel embarrassed to ask this because I probably overlooked something in the README, but I've genuinely tried to find an answer.)_

When I create a new note, the filename is assigned a unique ID, and the actual title of the note is added as a `# Title` and as an alias. This is different from what I'm accustomed to in native Obsidian. My basic issue now is: how do I search for notes by titles/aliases? I could use `:ObsidianSearch`, but that simply searches through all the content in the files.

I'm trying to understand the intended workflow. Having a unique ID as the filename prevents me from using my regular `find_files` feature in Telescope, so how do people manage this? If there is any documentation on this topic, I would be very grateful to be directed to it.

**Comment by epwalsh** [comment, marked as answer]
Hey @EddieBrunstedt, `:ObsidianSearch` is the preferred way, though if you want `find_files` to be more useful I would suggest tweaking the `note_id_func` in your obsidian.nvim config. You could set that function to only the title as the filename, for example.
